x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlMappingsUpgradeIT.java

```diff
@@ -105,8 +120,40 @@ private void assertUpgradedMappings() throws Exception {
             assertNotNull(propertiesLevel);
             // TODO: as the years go by, the field we assert on here should be changed
             // to the most recent field we've added that is NOT of type "keyword"
-            Map<String, Object> fieldLevel = (Map<String, Object>) propertiesLevel.get("multi_bucket_impact");
-            assertEquals(Collections.singletonMap("type", "double"), fieldLevel);
+            Map<String, Object> objectLevel = (Map<String, Object>) propertiesLevel.get("model_size_stats");
```

Comment:
Could using `XContentMapValues::extractValue` method make this more concise?

Reply:
Yes, good point.  I switched over to that.